.github/workflows/build_packages.yml

```diff
@@ -22,6 +22,29 @@ jobs:
     runs-on: ubuntu-22.04
     permissions:
       contents: write
+    env:
+      OUTPUT_DIR: "${{ github.workspace }}/bindist"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # Ubuntu packages.
+          - runs-on: ubuntu-24.04
+            platform: linux-x86_64
+            package: wave-lang
+            python-version: cp310-cp310
+          - runs-on: ubuntu-24.04
+            platform: linux-x86_64
+            package: wave-lang
+            python-version: cp311-cp311
+          - runs-on: ubuntu-24.04
+            platform: linux-x86_64
+            package: wave-lang
+            python-version: cp312-cp312
+          - runs-on: ubuntu-24.04
+            platform: linux-x86_64
+            package: wave-lang
+            python-version: cp313-cp313
```

Comment:
Quick comment: You can build for different Python versions in the `build_tools/build_linux_package.sh` script:
```
PYTHON_VERSIONS="${OVERRIDE_PYTHON_VERSIONS:-cp310-cp310 cp311-cp311 cp312-cp312 cp313-cp313}"
```
No need to span a matrix here.

Reply:
thanks, will remove from here.

Reply:
I was actually wrong and you want to spin the matrix here to build for different Python versions in parallel.